frontend/src/metabase/query_builder/actions/downloading.ts

```diff
@@ -127,15 +127,20 @@ const getDatasetParams = ({
   };
 };
 
+// remove trailing slash
+const BASENAME = (window.MetabaseRoot ?? "").replace(/\/+$/, "");
```

Comment:
Why not use `api.basename` here?

Reply:
because we do not import `api` here and I didn't want to bring such overhead to get just basename. Anyhow, let me change this

Reply:
fixed